web/styles/popovers.css

```diff
@@ -1224,41 +1202,43 @@ ul {
     }
 }
 
-.navbar-dropdown-menu {
+.popover-menu {
     margin: 0;
-    border: solid 1px var(--color-border-dropdown-menu);
-    background-color: var(--color-background-dropdown-menu);
     max-height: 85vh;
     overflow-x: hidden;
     user-select: none;
     border-radius: 6px;
-    box-shadow: var(--box-shadow-navbar-dropdown-menu);
 
     .simplebar-content {
         min-width: var(--popover-menu-min-width);
-        /*  This is necessary to set the dropdown menu width equal to
-            the width of the longest menu item, thus letting the menu
-            items control the width of the menu. */
+        /* Set the popover menu width equal to the width of the
+        longest menu item, thus letting the menu items control
+        the width of the menu. */
         width: min-content;
```

Comment:
I would think this would be `max-content` here, not `min-content`? But I think it's also possible that the `min-width` and `max-width` declarations here negate the `width` value anyway?

Reply:
All right :) I think I understand--thanks for the visualization!

Reply:
I'm glad it helped, thanks for taking your time to read through it! :)